### DISCUSSION TITLE
Unable to Book the exam slot for GitHub exam

### DISCUSSION THREAD

**Discussion by BeatByteCode** [community/community#128925] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,
I am attempting to schedule the exam, but I consistently encounter the following error when logging into the PSI exam portal. Can anyone help?
![image](https://github.com/community/community/assets/60009241/c3e6d546-b209-4b30-8e2c-ad8fc5ff69f9)

**Comment by ebndev** [comment]
Hi @BeatByteCode,

A 500 error indicates an internal error with the service provider. You should contact the person who gave you the link to the exam or check if PSI is experiencing outages.

It's possible that another GitHub user might have run into this same issue and can help, but we want to make sure you’re getting the best support you can and this space may not be the right place.

Best of luck!
